### ISSUE TITLE
Modal Accessiblity

### ISSUE THREAD

**Issue by davodey** [davodey/pearson-glp-pl#75]
Per @pauljadam suggestions:

There are other accessibility properties that can be added to the modal dialog like aria-describedby reference to the dialog text, aria 1.1. added aria-modal=true, ARIA authoring practices list out the attributes and keyboard operation, http://w3c.github.io/aria-practices/#dialog_modal.

One other part that should probably be there is hiding the visually grayed out content when the modal appears so the screen reader and keyboard user can't focus on that grayed out content. This is done with aria-hidden and removing keyboard focus. I have a simple demo but it's missing aria-modal http://pauljadam.com/demos/dialog.html

**Comment by antelopeb** [comment]
Also should include the ability to escape key out, or click on the background.
